frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -221,7 +221,7 @@ const Profile: NextPage = () => {
       </>
     ) : (
       <>
-        <a className={styles["open-button"]} href="#mpp-choose-subdomain">
+        <a className={styles["open-button"]} href="/premium#pricing">
```

Comment:
This changes the link whenever the user doesn't have a subdomain set, but that might also be the case for someone who _does_ have Premium, in which case this link and the lock icon (and maybe the wording, but that's already present in the current website) don't make much sense:

![image](https://github.com/mozilla/fx-private-relay/assets/4251/2123c17e-f3e4-4558-9b18-abbe1af1347e)

Reply:
Thanks for catching that!